homeassistant/components/bring/__init__.py

```diff
@@ -38,24 +38,26 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         await bring.login()
         await bring.load_lists()
     except BringRequestException as e:
+        _LOGGER.error("Failed to connect to server")
```

Comment:
```suggestion
```

We raise, so it seem to me that no additional logging is needed

Reply:
removed the lines